File: 01919.py
alpha1 = [0] * 26
alpha2 = [0] * 26

s1 = input()
for i in range(len(s1)):
    alpha1[ord(s1[i]) - ord('a')] += 1

s2 = input()
for i in range(len(s2)):
    alpha2[ord(s2[i]) - ord('a')] += 1

cnt = 0
for i in range(26):
    if alpha1[i] != alpha2[i]:
        cnt += abs(alpha1[i] - alpha2[i])

print(cnt)

File: 01919.cpp
#include <iostream>
#include <cmath>
using namespace std;

int alpha1[26];
int alpha2[26];

int main() {
	string s1; cin >> s1;
	for (int i = 0; i < s1.length(); i++)
		alpha1[s1[i] - 'a']++;

	string s2; cin >> s2;
	for (int i = 0; i < s2.length(); i++)
		alpha2[s2[i] - 'a']++;

	int cnt = 0;
	for (int i = 0; i < 26; i++)
		if (alpha1[i] != alpha2[i])
			cnt += abs(alpha1[i] - alpha2[i]);

	cout << cnt << endl;
}
